documentation/src/main/asciidoc/user_guide/chapter-63-Streams.adoc

```diff
@@ -0,0 +1,414 @@
+==  Streams
+
+Java 8 introduced the concept of a link:https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html[Stream]
+which allows defining operations to perform on your data rather than having to iterate and perform them both.
```

Comment:
which allows functional-style operations on collections rather than having to procedurally iterate over the data yourself.